app/models/Ticker.scala

```diff
@@ -33,13 +16,9 @@ object TickerName {
 
 case class TickerCopy(
   countLabel: String,
-  goalReachedPrimary: Option[String] , //deprecated
-  goalReachedSecondary: Option[String] //deprecated
 )
 
 case class TickerSettings(
-  endType: TickerEndType,
-  countType: TickerCountType,
```

Comment:
`endType` and `countType` are unused by the clients, but I think this may cause validation to fail in SDC and DCR as they're still required there -
https://github.com/guardian/support-dotcom-components/blob/main/src/shared/types/props/shared.ts#L101

I think your options are:
1. first remove them in SDC (and update the dependency in DCR)
2. leave them in for now and tidy up later

Reply:
PR  to remove endType and countType from TickerSettings  in SDC https://github.com/guardian/support-dotcom-components/pull/1335